AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminSaveAdobeStockLicensedImageActionGroup.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSaveAdobeStockLicensedImageActionGroup">
+        <waitForElementVisible selector="{{AdminAdobeStockSaveImagePreviewSection.name}}" time="30" stepKey="waitForInputVisible"/>
+        <click selector="{{AdminAdobeStockImagePreviewSection.confirm}}" stepKey="confirmSavingLicensed"/>
+        <waitForPageLoad stepKey="waitForImageToSave"/>
```

Comment:
It would be better to waitForLoadingMaskToDisappear

Reply:
Well spotted, thanks @Nazar65